libs/utils/WordPressUtils/src/test/java/org/wordpress/android/util/LogFileCleanerTest.kt

```diff
@@ -0,0 +1,70 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import java.io.File
+import java.io.FileReader
+import kotlin.random.Random
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.wordpress.android.util.helpers.logfile.LogFileCleaner
+import org.wordpress.android.util.helpers.logfile.LogFileProvider
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.O_MR1])
+class LogFileCleanerTest {
+    lateinit var logFileProvider: LogFileProvider
+    private val maxFiles = 10
```

Comment:
We can make this a constant and move it outside the class:

```
private const val MAX_FILES = 10
```

Reply:
Sounds good – changed in 3185b16a1144876ede635a895798f1b6663ed11e